src/index.ts

```diff
@@ -1 +1,52 @@
-// TODO: Execution starts here
+#!/usr/bin/env node
+
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+import yargs from 'yargs';
+import {Runner} from './runner';
+
+const args = yargs.options({
+  p: {
+    alias: 'project',
+    demandOption: true,
+    default: 'tsconfig.json',
+    describe: 'Path to TypeScript config file',
+    type: 'string',
+  },
+  m: {
+    alias: 'mode',
+    demandOption: true,
+    default: 'all',
+    describe:
+      "Choose between only leaving comments ('comment') or both comments and mutative fixes ('all')",
+    type: 'string',
```

Comment:
enum?

Reply:
yargs doesn't support enum options, but it does have a `choices` functionality that limits the possible acceptable inputs and displays it in the help, so I used that instead